pages/_app.js

```diff
@@ -1,10 +1,12 @@
-import { useState } from "react";
 import GlobalStyle from "../styles";
 
 import { mockSpots } from "../helpers/mockSpots.js";
+import useLocalStorage from "@/helpers/localStorage";
 export default function App({ Component, pageProps }) {
-  const [spots, setSpots] = useState(mockSpots);
-
+  const [spots, addSpots] = useLocalStorage("spots", mockSpots);
```

Comment:
To be consistent with state setter actions, I would call it `setSpots` here, as you had it before.